docs/tarball.md

```diff
@@ -0,0 +1,170 @@
+# Install Percona Distribiution for PostgreSQL from binary tarballs
+
+You can find the binary tarballs on the [Percona website](https://www.percona.com/downloads). Select the desired version from a version dropdown and _All_ from the Select Platform dropdown.
+
+There are the following tarballs available: 
+
+* percona-postgresql-{{dockertag}}-ssl1.1-linux-x86_64.tar.gz  - for operating systems that run OpenSSL version 1.x
+* percona-postgresql-{{dockertag}}-ssl3-linux-x86_64.tar.gz - for for operating systems that run OpenSSL version 3.x
+
+To check what OpenSSL version you have, run the following command: 
+
+```{.bash data-prompt="$"}
+$ openssl version
+```
+
+## Tarball contents
+
+The tarballs include the following components:
+
+| Component | Description |
+|-----------|-------------|
+| percona-postgresql{{pgversion}}| A metapackage that installs the latest version of PostgreSQL. It also includes the following extensions: <br> - `pgaudit` <br> - `pgAudit_set_user` <br> - `pg_repack` <br> - `pg_stat_monitor` <br> - `pg_gather` <br> - `wal2json` <br> -  the set of [contrib extensions](contrib.md)|
```

Comment:
percona-postgresql{{pgversion}} is not a meta package in case of tarball. It is in fact complete PostgreSQL server with its extensions.